pkg/watermark/otbucket/otbucket.go

```diff
@@ -0,0 +1,33 @@
+package otbucket
```

Comment:
```suggestion
// Package ot ....doc....
package otbucket
```

assuming you change the name to `ot`